### ISSUE TITLE
CR Snapshot Update Request for Decentralized Identifiers (DIDs) v1.0

### ISSUE THREAD

**Issue by iherman** [w3c/transitions#343]
# Document URLs

Decentralized Identifiers (DIDs) v1.0
- Document on TR: https://www.w3.org/TR/2021/CR-did-core-20210615/ (the planned URI)
- Editor's draft: https://w3c.github.io/did-core/
- Static version of the editors' draft: https://w3c.github.io/did-core/CR/2021-06-15/
- Expected publication date: 2021-06-15

# Link to group's decision to request transition

https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-06-08-did#resolution1

# Link to previous Candidate Recommendation transition or update request

https://github.com/w3c/transitions/issues/320

# Substantive changes

No substantive changes.

The purpose of this CR2 is to mark two features as at risk so that we may remove them at the end of CR2 if we don't get enough implementation support. The group was expecting two specific features to be implemented and was surprised when no implementers stepped forward to implement them during CR1.
We did also implement changes that were already covered by at risk issue markers during CR1 but are not considering those as substantive changes since they were clearly identified before entering CR1.

# Any changes in requirements?

None.

# Wide Review of substantive changes

While there are no substantive changes, the group is pre-emptively marking two features as at risk that are at reasonable risk of missing the exit criteria of two Independent implementations by the end of the Candidate Recommendation phase.

The conversation on the mailing list is here:

https://lists.w3.org/Archives/Public/public-did-wg/2021May/0023.html

The relevant telecon conversations are here:

https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-05-25-did#section1
https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-06-01-did#section5

# Issues status

All issues that remain, not counting at risk issue markers, are editorial in nature. The following is the complete list of remaining issues:

- Diagrams need SVG and detailed description - https://github.com/w3c/did-core/issues/625
- Verification method IDs MUST contain a fragment - https://github.com/w3c/did-core/issues/708
- FAQ Question: Can the DID Subject of a DID Document be updated/changed after the DID Document has been registered? - https://github.com/w3c/did-core/issues/718
- FAQ Question: Can the DID Controller of a DID Document be updated/changed after the DID Document has been registered? - https://github.com/w3c/did-core/issues/719
- All appendix sections needs to be cleaned up. - https://github.com/w3c/did-core/issues/728 
- When is the DID URI getting constructed - https://github.com/w3c/did-core/issues/729

# Formal Objections

None

# Any changes in implementation information?

- Since CR1, 18 implementations have been submitted to the conformance test suite. 
- Save for the new at risk issue markers for CR2, all features are either implemented by two independent implementations (the majority of the features in the specification are implemented), or marked as at risk.
- Minimum CR duration is 28 days. We expect all remaining implementations we are waiting on to be submitted in the next few weeks, before the end of CR2.
- Exit criteria remain the same - two independent implementations for each normative statement in the specification.

# Deadline for further comments

2021-07-13

# Any changes in patent disclosures?

None

cc @msporny @burnburn @brentzundel

**Comment by swickr** [comment]
Transition approved.

(CR Snapshot Update)

**Comment by iherman** [comment]
Official [publication request](https://lists.w3.org/Archives/Team/webreq/2021Jun/0022.html), plus the draft AC news item and draft HPN news item sent to the Webmaster, resp. the comm team.

**Comment by iherman** [comment]
The publication has been successfully done: https://lists.w3.org/Archives/Group/group-did-wg-chairs/2021Jun/0013.html

**Comment by iherman** [comment]
@plehegar I presume you are the one who has the right of closing this issue...
